src/ProcessManager/SendTodoDeadlineExpiredMailSubscriber.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace Prooph\ProophessorDo\ProcessManager;
+
+use Prooph\ProophessorDo\Model\Todo\Event\TodoWasMarkedAsExpired;
+use Prooph\ProophessorDo\Projection\Todo\TodoFinder;
+use Prooph\ProophessorDo\Projection\User\UserFinder;
+use Zend\Mail\Transport\TransportInterface;
+use Zend\Mail;
+
+/**
+ * Class SendTodoDeadlineExpiredMailSubscriber
+ *
+ * @package Prooph\ProophessorDo\App\Mail
+ * @author Michał Żukowski <michal@durooil.com
+ */
+final class SendTodoDeadlineExpiredMailSubscriber
+{
+    /**
+     * @var UserFinder
+     */
+    private $userFinder;
+
+    /**
+     * @var TodoFinder
+     */
+    private $todoFinder;
+
+    /**
+     * @var TransportInterface
+     */
+    private $mailer;
+
+    /**
+     * SendTodoDeadlineExpiredMailSubscriber constructor.
+     * @param UserFinder $userFinder
+     * @param TodoFinder $todoFinder
+     * @param TransportInterface $mailer
+     */
+    public function __construct(
+        UserFinder $userFinder,
+        TodoFinder $todoFinder,
+        TransportInterface $mailer
+    ) {
+        $this->userFinder = $userFinder;
+        $this->todoFinder = $todoFinder;
+        $this->mailer = $mailer;
+    }
+
+    /**
+     * @param TodoWasMarkedAsExpired $event
+     * @return void
+     */
+    public function __invoke(TodoWasMarkedAsExpired $event)
+    {
+        $todo = $this->todoFinder->findById($event->todoId()->toString());
+        $user = $this->userFinder->findById($todo->assignee_id);
+
+        $message = sprintf(
+            'Hi %s! Just a heads up: your todo `%s` has expired on %s.',
+            $user->name,
+            $todo->text,
+            $todo->deadline
+        );
+
+        $mail = new Mail\Message();
```

Comment:
Yeah ok, we can do this with another PR.

About the namespace. I'm not 100% sure, but something like Prooph\ProophessorDo\Mail\Command\* or maybe we just leave it in the user or todo namespace? What would you think yourself?